lib/utils.nix

```diff
@@ -11,6 +11,9 @@ with lib;
     "__empty" = null;
   };
 
+  # Turns all the keys of an attrs to raw lua.
+  rawKeysAttrs = mapAttrs' (n: v: nameValuePair "__rawKey__${n}" v);
```

Comment:
```suggestion
  toRawKeysAttrs = mapAttrs' (n: v: nameValuePair "__rawKey__${n}" v);
```
?

Reply:
Agreed. I'm unsure on `toRawKeysAttrs` vs `toRawKeyAttr` though

Reply:
Hmm, maybe `toRawKeys` ? X) I can't choose.

Reply:
I think either `toRawKeys`, `toRawKeyAttrs` (one `s` total), or `attrsToRawKeys`.

We could also have a companion function `mkRawKey = key: value: toRawKey { ${name} = value; }`